guides-and-tutorials/running-a-signer/signer-quickstart.md

```diff
@@ -0,0 +1,548 @@
+# Signer Quickstart
+
+If you want to get up and running as an active signer as quickly as possible. Here is a list of the commands you need to run and actions to take.
+
+If you are not familiar with how signing works yet, be sure to check out the [Stackers and Signing](/concepts/block-production/stackers-and-signing.md) concept guide.
+
+If you would like a more detailed walkthrough of all of these steps, take a look at the [Running a Signer](/guides-and-tutorials/running-a-signer/README.md) guide.
+
+## Step 1 - Prerequisites
+
+{% tabs %}
+
+{% tab title="Mainnet" %}
+
+```bash
+# Create the required directories
+mkdir -p ~/stacks-signer/data
+mkdir -p ~/stacks-node/data
+
+# Install needed packages
+sudo apt install -y npm wget unzip jq tar
+
+# Install Stacks CLI globally
+npm install --global @stacks/cli
+
+# Generate a new account and store details in a file
+stx make_keychain | jq > ~/stacks-signer/keychain.json
+```
+
+{% endtab %}
+
+{% tab title="Testnet" %}
+
+```bash
+# Create the required directories
+mkdir -p ~/stacks-signer/data
+mkdir -p ~/stacks-node/data
+
+# Install needed packages
+sudo apt install -y npm wget unzip jq tar
+
+# Install Stacks CLI globally
+npm install --global @stacks/cli
+
+# Generate a new account and store details in a file
+# '-t' option makes this a testnet account
+stx make_keychain -t | jq > ~/stacks-signer/keychain.json
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+The account file previously created looks like this:
+
+```json
+{
+  "mnemonic": "aaa bbb ccc ddd ...",
+  "keyInfo": {
+    "privateKey": "65f3...",
+    "publicKey": "03a3...",
+    "address": "SP1G...",
+    "btcAddress": "19tg...",
+    "wif": "Kzdt...",
+    "index": 0
+  }
+}
+```
+
+From this file, you'll need the `privateKey` value.
+
+## Step 2 - Set Up Your Stacks Signer
+
+### Download the stacks-signer binary
+
+Official binaries are available from the [Stacks Core releases page on Github](https://github.com/stacks-network/stacks-core/releases). Each release includes pre-built binaries. Download the [latest signer release ZIP file](https://github.com/stacks-network/stacks-core/releases/tag/signer-2.5.0.0.5.2) for your server’s architecture and decompress it. Inside of that folder is a `stacks-signer` binary. Currently, the latest mainnet and testnet versions that you have to use differ.
+
+Assuming a `Linux x64` machine, the commands to download and uncompress the signer binary look like this:
```

Comment:
I would be more specific here and mention glibc (if a user has a musl sytem, the links will not work for them). 
counterpoint is that users using musl by default will generally know they are and recognize the link won't work for them. 

i would default to being clear these are for glibc systems